COTL_API/CustomInventory/CustomCrops/CustomCropManager.cs

```diff
@@ -0,0 +1,99 @@
+using COTL_API.Guid;
+using HarmonyLib;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.ResourceManagement.AsyncOperations;
+using static UnityEngine.Object;
+
+namespace COTL_API.CustomInventory;
+
+public static partial class CustomItemManager
+{
+    private static GameObject s_cropPrefab = null!;
```

Comment:
line 12 `private static GameObject s_cropPrefab = null!;`

You may change this into an internal static if it is only for the api to use
The variable name can also be standardized such as CropPrefab